engine/crates/engine-v2/query-planning/src/graph.rs

```diff
@@ -37,14 +41,14 @@ pub trait Operation: std::fmt::Debug {
 
 pub struct OperationGraph<'ctx, Op: Operation> {
     pub(crate) schema: &'ctx Schema,
-    pub(crate) operation: &'ctx mut Op,
+    pub(crate) operation: Op,
     root_ix: NodeIndex,
     pub(crate) graph: StableGraph<Node<Op::FieldId>, Edge>,
 }
 
 impl<'ctx, Op: Operation> OperationGraph<'ctx, Op> {
     #[instrument(skip_all)]
```

Comment:
I know this is not part of this pull request, but if we use instrumentation like we do here, should we enforce them to be in debug level only? I haven't seen this span in traces, but wouldn't be surprised if we had a random `new` appearing in a customer demo...

Reply:
hum ok, fair enough